code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -2002,4 +2002,19 @@
 /datum/chemical_reaction/deuterium/on_reaction(var/datum/reagents/holder, var/created_volume)
 	var/turf/T = get_turf(holder.my_atom)
 	if(istype(T)) new /obj/item/stack/material/deuterium(T, created_volume)
-	return
\ No newline at end of file
+	return
+
+//Skrellian crap.
+/datum/chemical_reaction/talum_quem
+	name = "Talum-quem"
+	id = "talum_quem"
+	result = "talum_quem"
+	required_reagents = list("space_drugs" = 2, "sugar" = 1, "amatoxin" = 1)
+	result_amount = 4
+
+/datum/chemical_reaction/qerr_quem
+	name = "Qerr-quem"
+	id = "qerr_quem"
+	result = "qerr_quem"
+	required_reagents = list("nicotine" = 1, "carbon" = 1, "sugar" = 2)

```

Comment:
Nicotine isn't available anywhere, so this drug isn't mixable.

Reply:
I'll adjust that, then.

Reply:
Actually, I'll just leave it as unmixable for now- I'll make nicotine available in my planned E-Cigs PR.

Reply:
Let's not do that. Make the drug you're adding mixable, please.

Reply:
Alright.

Reply:
Added nicotine to cigarettes so you can harvest it for the recipes.